docs/pages/versions/unversioned/sdk/map-view.md

```diff
@@ -135,3 +136,24 @@ To use this in web, add the following script to your **web/index.html**. This sc
   <!-- <body /> -->
 </html>
 ```
+
+## How to retrieve your debug keystore fingerprint (Android only)
+
+When building a debug version of your application outside of Expo Go (for example, when using a [development client](https://docs.expo.dev/development/introduction/) or a standalone debug build), your app will be signed with the debug keystore on Android. The debug keystore location and password is defined in your `android/app/build.gradle` file like this:
+```groovy
+signingConfigs {
+    debug {
+        storeFile file('debug.keystore')
+        storePassword 'android'
+        keyAlias 'androiddebugkey'
+        keyPassword 'android'
+    }
+}
+``` 
+You can view the fingerprint for this keystore using the `keytool` command, and entering the storePassword. Copy the value shown after `SHA1:`
+```bash
+❯ keytool -list -v -keystore ./android/app/debug.keystore
+Enter keystore password: 
+```
+
+> If you are using one of the standard Expo templates, you don't need to perform these steps. All default templates use a debug keystore with fingerprint `5E:8F:16:06:2E:A3:CD:2C:4A:0D:54:78:76:BA:A6:F3:8C:AB:F6:25`, that you can enter directly in the Google Cloud Credential Manager.
```

Comment:
Let's move this up to line 141 so people will see this shortcut right away rather than first reading through a bunch of directions they might not need (even though they are very clearly and nicely explained!)

Reply:
Thanks. Moved it up between the explanation and the actual steps, and rephrased it a bit to fit there better.